packages/app/src/systems/Transaction/components/ActivityItem/ActivityItem.tsx

```diff
@@ -0,0 +1,140 @@
+import { cssObj } from '@fuel-ui/css';
+import { Card, Copyable, Flex, Icon, Text } from '@fuel-ui/react';
+import { bn } from 'fuels';
+import type { FC } from 'react';
+import { useMemo } from 'react';
+
+import type { Tx } from '../../utils';
+import {
+  OperationDirection,
+  getOperationDirection,
+  getTxStatusColor,
+} from '../../utils';
+import { TxIcon } from '../TxIcon';
+
+import { ActivityItemLoader } from './ActivityItemLoader';
+
+import { getAssetInfoById } from '~/systems/Asset';
+import { shortAddress } from '~/systems/Core';
+
+export type TxItemProps = {
+  transaction: Tx;
+  ownerAddress: string;
+};
+
+type TxItemComponent = FC<TxItemProps> & {
+  Loader: typeof ActivityItemLoader;
+};
+
+export const ActivityItem: TxItemComponent = ({
+  transaction,
+  ownerAddress,
+}) => {
+  const {
+    status: txStatus,
+    id = '',
+    totalAssetsSent,
+    operations,
+    time,
+  } = transaction;
+  const txColor = getTxStatusColor(txStatus);
+
+  const mainOperation = operations[0];
+  const label = mainOperation.name;
+  const amount = totalAssetsSent[0];
+  const date = time ? new Date(time) : null;
+
+  const toOrFromText = useMemo(() => {
+    const opDirection = getOperationDirection(mainOperation, ownerAddress);
+    if (opDirection === OperationDirection.to) return 'To: ';
+    if (opDirection === OperationDirection.from) return 'From: ';
+    return '';
+  }, [ownerAddress, mainOperation]);
+
+  const toOrFromAddress = useMemo(() => {
+    const opDirection = getOperationDirection(mainOperation, ownerAddress);
+    if (opDirection === OperationDirection.to)
+      return mainOperation.to?.address || '';
+    if (opDirection === OperationDirection.from)
+      return mainOperation.from?.address || '';
+    return '';
+  }, [ownerAddress, mainOperation]);
+
+  const formatDate = (date: Date) =>
+    `${date.toLocaleString('default', { month: 'short' })} ${date.getDate()}`;
+
+  const assetInfo = getAssetInfoById(amount?.assetId, amount);
+
+  return (
+    <Card css={styles.root} data-testid="activity-item">
+      <TxIcon operationName={mainOperation.name} />
+      <Flex direction="column" css={styles.contentWrapper}>
```

Comment:
You can use the `<Stack>` component from `@fuel-ui` instead of `<Flex direction="column">` here